addon/components/nrg-search.js

```diff
@@ -185,5 +185,6 @@ export default class NrgSearchComponent extends NrgValidationComponent {
   clear() {
     this.searchString = null;
     this.onChange(null);
```

Comment:
Does this not satisfy the needs for `onClear`? It will fire the `this.args.onChange` action with `null` as its argument.